api/_mixins/HTMLHyperlinkElementUtils__HTMLAreaElement.json

```diff
@@ -530,10 +530,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "10.1"
```

Comment:
This seems late. Where did the previous data come from and what did toString() do before 10.1?

Reply:
Before 10.1, the `toString()` method returns `[object HTMLAreaElement]`.  The previous data came from #12449, but that's my PR, and I think I messed up the testing somehow when I made that PR.

Reply:
OK, thanks for double checking!